examples/minesweeper/shared/build.gradle.kts

```diff
@@ -15,7 +15,16 @@ kotlin {
     jvm("desktop")
 
     ios()
-    iosSimulatorArm64()
```

Comment:
Maybe it's worth adding those flags to ios and macos targets too. 
Although, we will likely update to 1.5.0-beta soon, so it will be fixed automatically